core/models.py

```diff
@@ -61,7 +62,8 @@ def generate_unique_slug(self: "Community") -> str:
         return unique_slug
 
     def count_online_users(self: "Community") -> int:
-        online_limit = timezone.now() - timedelta(minutes=settings.LAST_ACTIVITY_ONLINE_LIMIT_MINUTES)

```

Comment:
Revert this changes please. Seems your editor is set to shorter lines than project settings, you need to adjust. 
Our line lenght is 120 chars: https://github.com/pymasterspl/reddit/blob/6810df3c26114963e2be001cad26822025886179/pyproject.toml#L33

The problem: this change breaks git history and makes debugging harder also CR longer. 

Please revert all those line length changes and re request review.